specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSubscription.json

```diff
@@ -400,244 +749,605 @@
+          "description": "Billing cycle details of the product.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/NextBillingCycleDetails"
+            }
+          ]
+        },
+        "offerId": {
+          "description": "The offer ID for the subscription. This field is only available for the Microsoft Online Services Program billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
+        "productCategory": {
+          "description": "Category of the subscription. Possible values include: ReservationOrder, SavingsPlanOrder, AzureSupport, UsageBased, Hardware, SaaS, SeatBased, and Software.",
+          "enum": [
+            "Other",
+            "ReservationOrder",
+            "SavingsPlanOrder",
+            "AzureSupport",
+            "UsageBased",
+            "Hardware",
+            "SaaS",
+            "SeatBased",
+            "Software"
+          ],
+          "type": "string",
           "readOnly": true,
-          "type": "string"
+          "x-ms-enum": {
+            "name": "BillingSubscriptionCategory",
+            "modelAsString": true
+          }
         },
-        "lastMonthCharges": {
-          "$ref": "./types.json#/definitions/Amount",
-          "description": "The last month charges.",
+        "productType": {
+          "description": "Type of the product for which the subscription is purchased.",
+          "type": "string",
           "readOnly": true
         },
-        "monthToDateCharges": {
-          "$ref": "./types.json#/definitions/Amount",
-          "description": "The current month to date charges.",
+        "productTypeId": {
+          "description": "Id of the product for which the subscription is purchased.",
+          "type": "string",
           "readOnly": true
         },
-        "reseller": {
-          "$ref": "#/definitions/Reseller",
-          "description": "Reseller for this subscription.",
+        "purchaseDate": {
+          "format": "date-time",
+          "description": "Purchase date of the product in UTC time.",
+          "type": "string",
           "readOnly": true
         },
+        "quantity": {
+          "format": "int64",
+          "description": "The number of licenses purchased for the subscription.",
+          "type": "integer"
+        },
+        "reseller": {
+          "description": "Details of the reseller.",
+          "allOf": [
+            {
+              "$ref": "./types.json#/definitions/Reseller"
+            }
+          ]
+        },
+        "renewalTermDetails": {
+          "description": "Details for the subscription's next renewal term.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/RenewalTermDetails"
+            }
+          ]
+        },
+        "skuId": {
+          "description": "The SKU ID of the product for which the subscription is purchased. This field is is only available  for Microsoft Customer Agreement billing accounts.",
```

Comment:
Same comment here as for productTypeId... we need to move to UPN (productCode).

Reply:
@ramkd12 @amberbhargava to comment here